src/test/java/uk/gov/hmcts/juror/api/moj/service/report/YieldPerformanceReportServiceImplTest_beforeEach_flow.puml

```diff
@@ -0,0 +1,83 @@
+@startuml
+
+/' diagram meta data start
+config=FlowConfiguration;
+{
+  "rootMethod": "uk.gov.hmcts.juror.api.moj.service.report.YieldPerformanceReportServiceImplTest#beforeEach()",
```

Comment:
What does file this do?

Reply:
not needed, have removed, its a plant UML file